unittests/wasm_config_tests.cpp

```diff
@@ -158,7 +158,7 @@ static const char many_exports_wast[] = R"=====(
   (func (export "apply") (param i64 i64 i64))
 )
 )=====";
-static const char one_export[] =  "(export \"fn${N}\" (func 0))";
+//static const char one_export[] =  "(export \"fn${N}\" (func 0))";
```

Comment:
why not remove?

Reply:
Because I thought it might be useful in the future. Since this is non-trivial, I thought I would keep it commented out.